go/roothash/tendermint/tendermint.go

```diff
@@ -242,6 +232,16 @@ func (tb *tendermintBackend) ComputeCommit(ctx context.Context, id signature.Pub
 	return nil
 }
 
+func (tb *tendermintBackend) ConsensusParameters(ctx context.Context, height int64) (*api.ConsensusParameters, error) {
+	/// XXX: could cache per height.
```

Comment:
Since this wont be changing at the moment, should we just cache the first result?

Reply:
Yeah we can cache it for now, just add a comment when this would need to change.